_posts/2020-11-10-covid-testing-sites-map-resources-relationships.md

```diff
@@ -0,0 +1,83 @@
+
+## Resources: ArcGIS Online
+
+We created the map and its underlying dataset quickly and have seen a massive increase in their usage since. The cloud-based ArcGIS Online infrastructure and tools allowed us to create them quickly and scale them easily and effortlessly. We didn't have to write any custom code or handle any storage, performance, or network issues, even though they have become some of our most-used resources. This is the same infrastructure behind Johns Hopkins' COVID dashboard that has been a key part of our collective understanding of the pandemic.
+
+### Web AppBuilder
+
+We built the map (or technically the application) using ArcGIS Online's Web AppBuilder. This WYSIWYG editor allowed us to configure a handful of widgets to get the needed functionality. It is embedded in the main site via an `<iframe>` to provided a (mostly) seamless integration with the rest of the site. It is functional on mobile devices, though not quite optimized for their smaller screens.
+
+### Hosted Feature Services
+
+For our [data source](https://utah.maps.arcgis.com/home/item.html?id=83b9c8ea89074aedbaea17cfbde98dec), we started with a copy of the nation-wide, volunteer-driven dataset being created by [GISCorps](https://covid-19-giscorps.hub.arcgis.com/pages/contribute-covid-19-testing-sites-data). We created our own copy, instead of updating the existing dataset directly, to allow us to manage the schema according to our needs and have more direct control over what sites are shown.
+
+Because our data source is a [hosted feature layer](https://doc.arcgis.com/en/arcgis-online/reference/feature-layers.htm) in ArcGIS Online, we can use a single, unified dataset for multiple purposes. We have a private view for editing that allows either us or the Dept. of Health to update the data or add new sites. We also have the public version (linked at the beginning of this section) used by the map and publicly available to anyone else who wants it. We don't have to worry about changes propagating through different copies.
+
+As a feature service, our data set can also be accessed programmatically via it's [REST](https://developers.arcgis.com/rest/) endpoint. This enabled Twenty Labs to scrape our data on a regular interval for their app, and has also allowed the coronavirus.utah.gov website folks from the Dept. of Technology Services  to create a [list-based view](https://coronavirus.utah.gov/covid-testing-locations-list) of the testing site data. They were able to put this interactive app together in just a few days because they can call the REST API directly with a simple HTTP request. They didn't have to learn any new frameworks, and they don't have to worry about maintaining their own version of the data.
```

Comment:
```suggestion
As a feature service, our data set can also be accessed programmatically via its [REST](https://developers.arcgis.com/rest/) endpoint. This enabled Twenty Labs to scrape our data on a regular interval for their app, and has also allowed the coronavirus.utah.gov website folks from the Dept. of Technology Services  to create a [list-based view](https://coronavirus.utah.gov/covid-testing-locations-list) of the testing site data. They were able to put this interactive app together in just a few days because they can call the REST API directly with a simple HTTP request. They didn't have to learn any new frameworks, and they don't have to worry about maintaining their own version of the data.
```